sql/driver/wire.go

```diff
@@ -80,14 +100,17 @@ func (d Datum) Value() (driver.Value, error) {
 		val = t.BytesVal
 	case *Datum_StringVal:
 		val = t.StringVal
+	case *Datum_DateVal:
+		val = Date{t.DateVal.GoTime()}
```

Comment:
OK, I've backed out the last commit. Will revisit when #2327 is implemented.

On Sun, Sep 20, 2015 at 9:52 PM, Ben Darnell <notifications@github.com>
wrote:

> In sql/driver/wire.go
> <https://github.com/cockroachdb/cockroach/pull/2565#discussion_r39938336>:
>
> > @@ -80,14 +100,17 @@ func (d Datum) Value() (driver.Value, error) {
> >  		val = t.BytesVal
> >  	case *Datum_StringVal:
> >  		val = t.StringVal
> > +	case *Datum_DateVal:
> > +		val = Date{t.DateVal.GoTime()}
>
> The client and server each have a time zone to be aware of; this only
> works when they are the same. I believe we will need #2327
> <https://github.com/cockroachdb/cockroach/issues/2327> to have a good
> time zone policy; until then the only consistent policy is to use UTC
> everywhere. Introducing new dependencies on the server's system timezone is
> a step in the wrong direction.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/2565/files#r39938336>.
>